content/document.js

```diff
@@ -63,7 +63,7 @@ function extractLocale(folder) {
   // locale as we prefer to spell it (e.g. 'pt-BR' not 'Pt-bR')
   if (!locale) {
     throw new Error(
-      `Unable to figure out locale from ${folder} with ${localeFolderName}`
+      `Unable to figure out locale from '${folder}' with '${localeFolderName}'`
```

Comment:
An unimportant change but as I was debugging I got this error thrown a couple of times and it's hard to read if the variables weren't "escaped".